linera-views/src/dynamo_db.rs

```diff
@@ -1311,25 +1321,13 @@ pub fn create_dynamo_db_common_config() -> CommonStoreConfig {
     }
 }
 
-/// Creates a configuration for tests
-#[cfg(with_testing)]
-pub async fn create_dynamo_db_test_config() -> DynamoDbStoreConfig {
-    let common_config = create_dynamo_db_common_config();
-    let use_localstack = true;
-    let config = get_config(use_localstack).await.expect("config");
-    DynamoDbStoreConfig {
-        config,
-        common_config,
-    }
-}
-
 /// Creates a basic client that can be used for tests.
 #[cfg(with_testing)]
 pub async fn create_dynamo_db_test_store() -> DynamoDbStore {
-    let store_config = create_dynamo_db_test_config().await;
+    let config = DynamoDbStore::get_test_config().await.expect("config");
```

Comment:
In a test, you might as well use `.unwrap()` instead of `.expect` with a tiny error message

Reply:
Ok.